File: trunk/java/src/it/grabz/grabzit/GenericResult.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package it.grabz.grabzit;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

/**
 *
 * @author Administrator
 */
@XmlRootElement(name="WebResult")
class GenericResult implements IMessageResult {
    @XmlElement(name="Result")
    private String result;
    @XmlElement(name="Message")
    private String message;

    @Override
    public String getMessage() {
        return message;
    }
    
    public String getResult() {
        return result;
    }
}

File: scraper/aspnet/GrabzIt.Scraper/Results/GenericResult.cs
using System;
using System.Xml.Serialization;

namespace GrabzIt.Scraper.Results
{
    [Serializable]
    [XmlType(TypeName = "WebResult")]
    public class GenericResult : IException
    {
        public string Result;

        public string Message
        {
            get;
            set;
        }

        public string Code
        {
            get;
            set;
        }

        public GenericResult()
        {            
        }
    }
}